docs/question.md

```diff
@@ -374,10 +387,53 @@ int m = 4;
 </pl-code>
 ```
 
-## Escaping `<markdown>` tags
+### Escaping `<markdown>` tags
 
 Under the hood, PrairieLearn is doing some very simple parsing to determine what pieces of a question to process as Markdown: it finds an opening `<markdown>` tag and processes everything up to the closing `</markdown>` tag. But what if you want to have a literal `<markdown>` or `</markdown>` tag in your question? PrairieLearn defines a special escape syntax to enable this. If you have `<markdown#>` or `</markdown#>` in a Markdown block, they will be rendered as `<markdown>` and `</markdown>` respectively (but will not be used to find regions of text to process as Markdown). You can use more hashes to produce different strings: for instance, to have `<markdown###>` show up in the output, write `<markdown####>` in your question.
 
+### Using a dollar sign ($) without triggering math mode
```

Comment:
IMO this should be nested under the new section added below, not these docs about `<markdown>`.

Reply:
[03c7acb](https://github.com/PrairieLearn/PrairieLearn/pull/11819/commits/03c7acb7ada2775ace82abad511cbe2ec47d5f77)